src/rpc/rpcquorums.cpp

```diff
@@ -375,6 +396,39 @@ UniValue quorum_sigs_cmd(const JSONRPCRequest& request)
         } else {
             return llmq::quorumSigningManager->AsyncSignIfMember(llmqType, id, msgHash);
         }
+    } else if (cmd == "verify") {
+        CBLSSignature sig;
+        if (!sig.SetHexStr(request.params[4].get_str())) {
+            throw JSONRPCError(RPC_INVALID_PARAMETER, "invalid signature format");
+        }
+
+        llmq::CQuorumCPtr quorum{nullptr};
+        if (request.params[5].isNull() || (request.params[5].get_str().empty() && !request.params[6].isNull())) {
+            int signHeight{-1};
+            if (!request.params[6].isNull()) {
+                signHeight = ParseInt32V(request.params[6], "signHeight");
+            }
+            // First check against the current active set
+            quorum = llmq::quorumSigningManager->SelectQuorumForSigning(llmqType, id, signHeight, 0);
+            if (!quorum) {
+                // Then check against the previous active set in case it changed recently
+                auto signOffset = Params().GetConsensus().llmqs.at(llmqType).dkgInterval;
```

Comment:
I don't love using auto and making the type unclear. Why not just say int?

Reply:
done in ee2944a